src/test/java/study/StringCalculator.java

```diff
@@ -0,0 +1,97 @@
+package study;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringCalculator {
+
+    int calculator(String str) {
+        String[] cal_element = str.split(" ");
+        int num1 = Integer.parseInt(cal_element[0]);
+
+        for(int i=1; i<cal_element.length-1; i+=2) {
```

Comment:
```suggestion
        for (int i = 1; i < cal_element.length - 1; i += 2) {
```

코드 작성 후 Reformat을 한번 돌려주시면 좋겠네요!

IntelliJ 등의 IDE에는 해당 기능을 단축키로 제공하는 경우가 많습니다.
IntelliJ를 사용중이시면 아래 내용을 한번 참고해보세요.

https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

Reply:
넵! IntelliJ 사용 중입니다. 좋은 기능 알려주셔서 감사합니다!